vasttools/query.py

```diff
@@ -380,12 +380,17 @@ def __init__(
                 "\nPlease address and try again."
             ))
 
-        data_available = self._check_data_availability()
-        if not data_available:
-            raise QueryInitError((
-                "Not all requested data is available! "
-                "Please address and try again."
-            ))
+        self.logger.info("Checking data availability...")
+        all_data_available = self._check_data_availability()
+        if all_data_available:
+            self.logger.info("All data available!")
+        else:
+            self.logger.critical(
+                "Not all requested data is available! See above for details."
+            )
+            self.logger.critical(
+                "Query will continue run, but proceed with caution."
+            )
```

Comment:
I don't see this as a `critical` error - that usually denotes something that will cause the task to completely crash out. Given that the idea is that this picks out the missing ones and allows the query to continue it should probably be at the `warning` level?
```suggestion
            self.logger.warning(
                "Not all requested data is available! See above for details."
            )
            self.logger.warning(
                "Query will continue run, but proceed with caution."
            )
```